### ISSUE TITLE
Styled components - block-closing-brace-space-after

### ISSUE THREAD

**Issue by ortigozamatias** [stylelint/stylelint#4155]
This warning gets fired when I have more than one styled component in the same file. Linting is started via 'stylelint' command.

`
 Expected single space after "}" (block-closing-backtick-space-after)
`

The configuration linting file, .stylelintrc, is as follows: 

```json
{
    "processors": [
        "stylelint-processor-styled-components"
    ],
    "extends": [
        "stylelint-config-styled-components"
    ],
    "rules": {
        "block-closing-brace-space-after": "always"
    }
}
```

Here is an example component: 

```javascript
import React, { memo } from 'react';
import styled from 'styled-components/macro';
import SidebarOption from '../SidebarOption';

const StyledSidebar = styled.div`
    background: ${({ theme }) => theme.colorSidebar};
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    width: 72px;
`; // warning here

const Logo = styled.div`
    align-items: center;
    background: ${({ theme }) => theme.colorSidebarLogo};
    display: flex;
    height: 64px;
    justify-content: center;
`; // warning here

const Options = styled.div`
    display: flex;
    flex-direction: column;
`;

const Sidebar = () => (
    <StyledSidebar>
        <Options>
            <Logo>logo</Logo>
            <SidebarOption />
        </Options>
        <Options>
            <SidebarOption />
        </Options>
    </StyledSidebar>
);

export default memo(Sidebar);

```

Version: 10.1.0

**Comment by evilebottnawi** [comment]
Why do you use `stylelint-processor-styled-components`? `stylelint` support css-in-js out of box without `processor`

**Comment by ortigozamatias** [comment]
> Why do you use `stylelint-processor-styled-components`? `stylelint` support css-in-js out of box without `processor`

I was just following instructions on https://www.styled-components.com/docs/tooling#stylelint

**Comment by evilebottnawi** [comment]
Looks this instructions are invalid, try to remove `stylelint-processor-styled-components`

**Comment by ortigozamatias** [comment]
Warnings have disappeared yet other rules are highlighted like `no-empty-first-line`. I guess that's what the processor avoids.

**Comment by evilebottnawi** [comment]
@ortigozamatias please clarify, do you have other problems? Show how we can reproduce them, thanks

**Comment by ortigozamatias** [comment]
I have updated .stylelintrc file according to @evilebottnawi and added another rule, which is `no-empty-first-line`. Thankfully, the warning regarding `block-closing-brace-space-after` have disappeared but other ones have suddenly appeared, all of which are related to the added rule.

Updated .stylelintrc
```json
{
    "extends": [
        "stylelint-config-styled-components"
    ],
    "rules": {
        "block-closing-brace-space-after": "always",
        "no-empty-first-line": true
    }
}
```


Updated warnings on example component
```javascript
import React, { memo } from 'react';
import styled from 'styled-components/macro';
import SidebarOption from '../SidebarOption';

const StyledSidebar = styled.div` // warning: Unexpected empty line
    background: ${({ theme }) => theme.colorSidebar};
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    width: 72px;
`;

const Logo = styled.div` // warning: Unexpected empty line
    align-items: center;
    background: ${({ theme }) => theme.colorSidebarLogo};
    display: flex;
    height: 64px;
    justify-content: center;
`;

const Options = styled.div` // warning: Unexpected empty line
    display: flex;
    flex-direction: column;
`;

const Sidebar = () => (
    <StyledSidebar>
        <Options>
            <Logo>logo</Logo>
            <SidebarOption />
        </Options>
        <Options>
            <SidebarOption />
        </Options>
    </StyledSidebar>
);

export default memo(Sidebar);

```


Those last warnings do not appear when adding this to the configuration file
```json
"processors": [ "stylelint-processor-styled-components" ]
```

That is what I was saying that those kind of errors were likely to be hidden when using the processor for styled-components.

**Comment by evilebottnawi** [comment]
hm, strange, should be no errors, /cc @hudochenkov

**Comment by hudochenkov** [comment]
This is correct `no-empty-first-line` is not intended to be used with CSS-in-JS, as it looks at the source. In case of CSS-in-JS every component considered source (everything between backticks). 

@ortigozamatias Just disable this rule because it's not applicable here.

**Comment by ortigozamatias** [comment]
I get it. Thanks @evilebottnawi, @hudochenkov!.

**Comment by evilebottnawi** [comment]
@hudochenkov maybe we should disable this rule for css-in-js to avoid bad DX?
